test/unit/uk/gov/hmrc/thirdpartyapplication/controllers/ApplicationControllerSpec.scala

```diff
@@ -79,7 +80,8 @@ class ApplicationControllerSpec extends UnitSpec with ScalaFutures with MockitoS
       mockCredentialService,
       mockSubscriptionService,
       config,
-      mockAuthConfig)
+      mockAuthConfig,
+      mockGatekeeperService)
```

Comment:
it would be good to add a test here for the new endpoint, which only allows you to delete standard sandbox apps